.github/workflows/alpine-release.yml

```diff
@@ -0,0 +1,34 @@
+name: Alpine Linux release
+on: 
+  push:
+    tags:
+      - v*
+env:
+  IMAGE_NAME: "opensmtpd"
+jobs:
+  build:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Alpine Linux (amd64 musl openssl)
+      run: docker build . --file ci/docker/Dockerfile.alpine-release --tag opensmtpd:alpine
+
+    - name: Login to ghcr.io
+      run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u $ --password-stdin
```

Comment:
the `-u $` seems strange, forgot to type the variable' name?

Reply:
I actually lifted that from Github's documentation. I will change to use the docker login action https://github.com/docker/login-action instead.

Reply:
Fixed.